sduploader/sdlocaluploader.go

```diff
@@ -0,0 +1,39 @@
+package sduploader
+
+import (
+	"io"
+	"os"
+)
+
+const logFileName string = "build.log"
```

Comment:
Result to a local discussion, we should use not to duplicate name.

Reply:
Fixed it.